aten/src/ATen/native/cudnn/Conv_v8.cpp

```diff
@@ -162,27 +161,27 @@ struct BenchmarkCache {
 std::mutex mutex;
```

Comment:
It looks like the mutex is not used at all now. Can the field in BenchmarkCache be removed?

Reply:
Yes, thanks!